builder/dockerfile/buildargs_test.go

```diff
@@ -37,7 +37,7 @@ func TestGetAllAllowed(t *testing.T) {
 		"ArgWithDefaultInDockerfile":          "fromdockerfile1",
 		"ArgNoDefaultInDockerfileFromOptions": "fromopt3",
 		"ArgFromMeta":                         "frommeta1",
-		"ArgFromMetaOverriden":                "fromdockerfile3",
```

Comment:
This changes make the `gofmt` fails, because **now** this `"fromdockerfile3"` is mis-aligned with the others.